www/src/components/landingPage/tweets/tweets.astro

```diff
@@ -1,61 +1,88 @@
+>
+  <div
+    class="flex flex-col items-start text-left order-first md:order-last mb-10"
+  >
+    <h2 class="text-2xl font-semibold text-t3-purple-200">
+      Loved by the community
+    </h2>
+    <h3 class="mt-2 text-5xl tracking-tight font-bold text-t3-purple-50">
+      What others have to say about create-t3-app
+    </h3>
+    <p class="mt-4 text-xl text-t3-purple-200 max-w-3xl">
+      Want to be listed here? Mention <a
+        href="twitter.com"
+        rel="noopener noreferrer"
+        class="text-t3-purple-100">@placeholder</a
+      > on Twitter!
+    </p>
+  </div>
+  <div class="swiper w-full rounded-lg">
+    <div class="swiper-wrapper hidden">
+      <div class="swiper-slide">
+        <Tweet
+          author="Emilia Zapata"
+          avatar="https://pbs.twimg.com/profile_images/1426010455055638531/xUFu5JP7_400x400.jpg"
+          comments={2}
+          handle="synecdokey"
+          likes={75}
+          retweets={9}
+          date="Jul 31"
+          text="Finally had the chance to play with @trpcio and it's scary how easy it makes to make your data flow, with full type safety and backend validation. With create-t3-app and @supabase, I was able to create a simple to-do app with auth in less than 3 hours, scary."
+          verified={false}
+          tweetId="1553580714591158272"
+        />
+      </div>
+      <div class="swiper-slide">
+        <Tweet
+          author="Anthony (ajcwebdev.x)"
+          avatar="https://pbs.twimg.com/profile_images/1549247631867711488/hK_Qr-Dx_400x400.png"
+          comments={9}
+          handle="ajcwebdev"
+          likes={18}
+          retweets={4}
+          date="Jul 7"
+          text="Now that Blitz.js has pivoted and Bison has stagnated, create-t3-app will be the only framework to give Redwood a run for its money in the quest to build a legitimate fullstack React framework."
+          verified={false}
+          tweetId="1544909672137867264"
+        />
+      </div>
     </div>
-    <div class="w-fit sm:p-2">
-      <Tweet
-        author="Anthony (ajcwebdev.x)"
-        avatar="https://pbs.twimg.com/profile_images/1549247631867711488/hK_Qr-Dx_400x400.png"
-        comments={9}
-        handle="ajcwebdev"
-        likes={18}
-        retweets={4}
-        date="Jul 7"
-        text="Now that Blitz.js has pivoted and Bison has stagnated, create-t3-app will be the only framework to give Redwood a run for its money in the quest to build a legitimate fullstack React framework."
-        verified={false}
-        tweetId="1544909672137867264"
-      />
+    <div class="mt-10">
+      <div class="swiper-pagination"></div>
     </div>
   </div>
-</section>
+</PageSection>
+
+<script type="module" is:inline client:visible>
+  import "../../../../node_modules/swiper/swiper-bundle.css";
+  import {
+    Autoplay,
+    Pagination,
+    Swiper,
+  } from "../../../../node_modules/swiper";
```

Comment:
These imports are probably why it doesn't work in deployments

Reply:
The imports weren't working properly. I refactored it into a react component and now it works as intended. Only thing left to fix is the pagination.

Reply:
Can you not use a normal import for the vanilla js lib?